### DISCUSSION TITLE
Can gofakeit set a unique country

### DISCUSSION THREAD

**Discussion by cdsjsz** [brianvoe/gofakeit#228] [Q&A]
Can I set a unique country name and then generate fake datas with that,such as phone number and state?
Thanks in advance.

**Comment by brianvoe** [comment]
First of all Gofakeit can do anything! Taxes? Yes! Bully bothering you in school? Use the power of Gofakeit random to take those bad guys out.

As far as your issue I have no idea what you're trying to accomplish. If you could help explain a little better what kind of data you want to use and what value you are expecting back(WITH CODE EXAMPLES!). Then ill help put something together for you.

**Reply by cdsjsz** [reply]
> First of all Gofakeit can do anything! Taxes? Yes! Bully bothering you in school? Use the power of Gofakeit random to take those bad guys out.
> 
> As far as your issue I have no idea what you're trying to accomplish. If you could help explain a little better what kind of data you want to use and what value you are expecting back(WITH CODE EXAMPLES!). Then ill help put something together for you.

Thx.I want to generate relevant data for a particular country, such as the state or mobile phone number in US.
I used to use python's [faker](https://github.com/joke2k/faker) library to implement it.The code is here:

from faker import Faker
fake = Faker( 'en_US')

Then I can get data only about US rather than random countries.Could you help me?

**Comment by brianvoe** [comment]
https://github.com/brianvoe/gofakeit/issues/196

I currently dont have any localization. Its something I have wanted to add but its a little harder than people think. Maybe one day this discussion will be updated with a new feature set.

**Reply by cdsjsz** [reply]
> #196
> 
> I currently dont have any localization. Its something I have wanted to add but its a little harder than people think. Maybe one day this discussion will be updated with a new feature set.

Oh,I got it.Thx for your efforts and looking forward to the feature.
